### PR TITLE
Adds HTML and starter content

### PR DESCRIPTION
**Pull request by kendrabu**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div><img src="images/desktop.jpg"></div>
+    <div>
+      <h1 id="section-title">Section-Title</h1>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Et magnis dis parturient montes nascetur ridiculus mus mauris vitae. Vitae purus faucibus ornare suspendisse. Nullam non nisi est sit amet facilisis. In ante metus dictum at. Pulvinar pellentesque habitant morbi tristique. Sed velit dignissim sodales ut eu. Diam ut venenatis tellus in metus vulputate eu. Cursus risus at ultrices mi tempus imperdiet nulla. Quis eleifend quam adipiscing vitae proin sagittis nisl rhoncus mattis. Enim lobortis scelerisque fermentum dui faucibus in ornare quam viverra. Odio facilisis mauris sit amet massa. Elit duis tristique sollicitudin nibh sit. Egestas congue quisque egestas diam in arcu cursus euismod quis. Porttitor leo a diam sollicitudin tempor.</p>
+
+      <h1 id="section-title-two">Section-Title-Two</h1>
```
Comment (axamei) [inline about.html]: There should only be 1 `<h1>` per page and they generally describe the content for the page. I recommend adding an `<h1>About</h1>` and changing the existing `<h1>` tags to `<h2>`. The pattern you have in your `index.html` is a good one to follow for headings.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Et magnis dis parturient montes nascetur ridiculus mus mauris vitae. Vitae purus faucibus ornare suspendisse. Nullam non nisi est sit amet facilisis. In ante metus dictum at.
+      </p>
+    </div>
+  </main>
+  <footer>
+ <p>@2019 Kendralyn's Portfolio</p>
```
Comment (axamei) [inline about.html]: You can use `&copy;` to get the copyright symbol.

**Review by axamei — CHANGES_REQUESTED** [review]

**Comment by kendrabu** [comment]
I have made these changes. Do they look correct?

File: home.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <li><a href="home.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="works.html">Works</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div><img src="images/desktop.jpg" alt="placeholder photo of desktop"></div>
+    <div>
+      <h3 id="section-title">Section-Title</h3>
```
Comment (axamei) [inline home.html:23]: You can't skip heading levels. The next one should be h2 and not h3. I am going to merge for now so you can move along and begin your CSS work.

**Review by axamei — COMMENTED** [review]
I am going to merge for now so you can move along and begin your CSS work.

File: home.html (hunk 3 above)
Comment (kendrabu) [reply home.html:23]: Thank you. I realized this was a typo on my part. Thank you for catching it.

**Pull request merged**